C-Plus-Plus/dp/Matrix_Chain_Multiplication.cpp

```diff
@@ -1,101 +1,69 @@
-// Matrix chain multiplication algorithm 
-#include <bits/stdc++.h> 
-using namespace std; 
+/*
+Question:- Given a sequence of matrices, find the most efficient way to multiple these matrices together. We have to find the way which costs minimum.
+How are we defining coat of matrix multiplication ?
+=> Let's asssume two matrices are there of order 10X30 and 30X20 so cost of multiplication of them will be 10*30*20 , we will take the common order element only once .
+*/
```

Comment:
Remove this and write
 - Add Multi-Line Comments 
- Add a beginner-friendly description at the top

Reply:
@ankitaggarwal23 changes done pls review